uds/can/transport_interface/python_can.py

```
@@ -815,9 +504,16 @@ def is_supported_network_manager(bus_manager: Any) -> bool:
 
         :param bus_manager: Value to check.
 
-        :return: True if provided bus object is compatible with this Transport Interface, False otherwise.
+        :return: True if provided bus object is python-can Bus object, False otherwise.
         """
-        return isinstance(bus_manager, BusABC)
+        if not isinstance(bus_manager, BusABC):
+            return False
+        network_manager_class_name = bus_manager.__class__.__name__
+        network_manager_module = bus_manager.__class__.__module__
+        for backend_name, (module_path, class_name) in BACKENDS.items():
```

Ruff (0.16.2)

[warning] Loop control variable `backend_name` not used within loop body

Rename unused `backend_name` to `_backend_name`

(B007)